chimney/src/test/scala/io/scalaland/chimney/DslSpec.scala

```diff
@@ -420,14 +420,14 @@ object DslSpec extends TestSuite {
     "support common data types" - {
 
       case class Foo(value: String)
-      case class Bar(value: String)
+      case class Bar(value: String, unit: Unit)
```

Comment:
I suggest to keep these test as they are. Instead, I would add new test case "support Unit" where you construct simple example with unit field in target case class.